app/views/application/_suggestion.html.erb

```diff
@@ -0,0 +1,26 @@
+<div class="pairing">
+  <div class="restaurant">
+    <h3>Restaurant</h3>
+    <%= link_to(restaurant.name, restaurant.url) %>
+    <%= image_tag restaurant.image_url %>
+    <%= restaurant.address.join('<br>').html_safe %>
+    <% if restaurant.phone %>
+    <p>Phone: <%= restaurant.phone.chars.insert(3, "-").insert(7, "-").join %></p>
```

Comment:
Try to remove most logic from the views and primarily keep them in controllers or models